### ISSUE TITLE
Add a page detailing the organisers

### ISSUE THREAD

**Issue by steadweb** [phpwarks/website#40]
Ideally, this page would contain a list / grid of us, with a short description on our strengths, goals etc and maybe what our job titles are? 

- Name
- Picture
- Handle (twitter)
- Bio

Thoughts @jedkirby, @NigelGreenway and @benplummer?

**Comment by NigelGreenway** [comment]
Yeah, good idea. Maybe create a branch and we add our own stuff as a commit each?

**Comment by jedkirby** [comment]
Agree with @NigelGreenway, need to be able to create a local instance of the site first, though.
